workspaces/refresh.sh

```diff
@@ -108,6 +116,20 @@ function refresh_sample_minimal_workspace {
 	copy_template theme-favicon sample-minimal-workspace/client-extensions/able-theme-favicon "Able Theme Favicon"
 	copy_template theme-spritemap sample-minimal-workspace/client-extensions/able-theme-spritemap "Able Theme Spritemap"
 
+	#
+	# Angular custom element client extension
+	#
+
+	rm -fr sample-minimal-workspace/client-extensions/angular-custom-element
+
+	cd sample-minimal-workspace/client-extensions
+
+	npx @angular/cli new angular-custom-element --defaults --skip-install
```

Comment:
How we could guarantee that committed patches can be applied in presence of newer `@angular/cli` releases? Shall we somehow fix the version we invoke to signal this compatibility? Or perhaps patch from the very beginning (i.e. include the angular app itself)?

Reply:
@lawrence-lee said that we were going to have QA tests for that, but it we want to make sure until that happens, we may fix Angular version.

Note also that, if patches cannot be applied, git will complain. Additionally, git is very lenient when patching, so as long as the new sample code resembles the old one, patches will work with high probability.

Reply:
I've been looking at `create_remote_app.sh` and [they don't use fixed version numbers](https://github.com/liferay/liferay-portal/blob/d6a9ec00ce6550a7da94751d637278935c058fc0/tools/create_remote_app.sh#L81). 

Additionally, I've found out that it not only creates React, but [Vue apps too](https://github.com/liferay/liferay-portal/blob/d6a9ec00ce6550a7da94751d637278935c058fc0/tools/create_remote_app.sh#L9).

Reply:
I like the idea of fixing our versions of tools we call in CLI scripts, otherwise I think it is pretty risky to just call the latest version.